### PR TITLE
issue-1-can-like-dislike - Dennis Liu

### PR DESCRIPTION
**Pull request by dennisliu9**
ajax-project_feature-1_solution1
ajax-project_feature-1_solution2

### CODE REVIEW

**Comment by dennisliu9** [comment]
Note: mobile view was changed from wireframes to make better use of vertical space.

**Comment by dennisliu9** [comment]
ajax-project_feature-1_solution3

Added recording that shows the placeholder image that appears on fresh load of app.

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        }
+      } else {
+        searchResultsIdx++;
+        handleSearchResponse(); // This will use the new value of searchResultsIdx since handleSearchResponse calls searchResultsIdx from a higher scope than itself
+      }
+      acquireRequest.removeEventListener('load', handleAcquireReponse);
+    });
+    acquireRequest.send();
+    searchRequest.removeEventListener('load', handleSearchResponse);
+  }
```
Comment (codingcodymiller) [inline js/main.js]: Is there any particular reason this needs to be inside of another function definition?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 1 above)
Comment (dennisliu9) [reply js/main.js]: Per our Slack discussion on variables (and their attached event listeners) being garbage collected, inner functions have been declared outside.

**Comment by dennisliu9** [comment]
Updated CSS for media queries:

Desktop_Landing
Desktop_Selection
Mobile_Landing
Mobile_Selection

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**